static/lineage.html

```diff
@@ -0,0 +1,59 @@
+<div class="row">
```

Comment:
Iirc, the lineage view should just be the old state view, renamed. There's no need for both.

Reply:
Oops my bad I forgot to remove it 😅